sn_networking/src/lib.rs

```diff
@@ -183,6 +184,16 @@ impl Network {
         self.get_closest_peers(key, false).await
     }
 
+    /// Returns the map of ilog2 distance of the Kbucket to the peers in that bucket
```

Comment:
the comment is bit confusing.
according to the return type, it seems the returned map is:
the map of the kbuckets table, indexing by the ilog2 distance of the kBucket to self ?

Also, if choosse the map to be `BTreeMap`, buckets will be `oredered` by itself ?

Reply:
Right! Thanks for the tip! Will update it!